teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, name = 'Unknown')
+    super(name)
```

Comment:
- [ ] Please make sure to pass the right parameters to the parent constructor like the following:
```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
```